requirements.txt

```diff
@@ -1,4 +1,4 @@
-pvlib>=0.7.0,<0.9.0
+pvlib>=0.7.0,<0.10.0
```

Comment:
👍

Reply:
@kanderso-nrel The thought popped into my head last night as to if it we should require pvlib 0.9.0 as a minimum, because of the error involved in the earlier versions.

(The error appears minor in the test you fixed here, but maybe that is a false impression and/or shouldn't matter.)